consensus/state.go

```diff
@@ -24,12 +24,17 @@ import (
 	"github.com/tendermint/tendermint/libs/service"
 	tmsync "github.com/tendermint/tendermint/libs/sync"
 	"github.com/tendermint/tendermint/p2p"
+	"github.com/tendermint/tendermint/pkg/remote"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 	sm "github.com/tendermint/tendermint/state"
 	"github.com/tendermint/tendermint/types"
 	tmtime "github.com/tendermint/tendermint/types/time"
 )
 
+const (
+	consensusTable = "consensus"
+)
```

Comment:
Is this used still?

Reply:
whoops! forgot to remove since I was still running tests using it good catch

https://github.com/celestiaorg/celestia-core/pull/970/commits/3e4a964fbcc647898ff886bc57036b54f940db72